class error(Exeption):
  pass

class error(Exeption):
  pass
